src/cli/commands/remove.js

```diff
@@ -41,13 +41,13 @@ export async function run(
       const registry = config.registries[registryName];
       const object = rootManifests[registryName].object;
 
-      for (const type of constants.DEPENDENCY_TYPES) {
```

Comment:
Actually this only happens only on Node 7 on ubuntu on windows.
Probably needs to be reported with Node.